apps/site/src/pages/_document.tsx

```diff
@@ -0,0 +1,21 @@
+import { Head, Html, Main, NextScript } from 'next/document';
+
+export default function Document() {
+  return (
+    <Html className="h-full" lang="en">
+      <Head>
+        <link rel="stylesheet" href="https://use.typekit.net/nys4xzx.css" />
```

Comment:
Can you try putting `<meta name="theme-color" content="#1a5d8d">` here (but change `content` to color that matches the header.

[So that it looks better on Safari]( https://useyourloaf.com/blog/safari-15-theme-color/)
![image](https://user-images.githubusercontent.com/71996763/189533092-e18f0c95-11e1-4821-a933-b5e03052a923.png)